app/controllers/profiles_controller.rb

```diff
@@ -38,7 +38,7 @@ def delete
   end
 
   def destroy
-    Delayed::Job.enqueue DeleteUser.new(current_user), priority: PRIORITIES[:profile_deletion]
+    DeleteUserJob.perform_later(user: current_user)
```

Comment:
Any reason to not pass just simple types as arguments (like `current_user.id`) in here? It make it easier to follow and more transparent. Otherwise there is some kind of serialization/deserialization happening. Current jobs are mostly using ids if I remember well.

Reply:
ActiveJob automatically serializes/deserializes records via global ids. no reason to do the `.id` and `.find` ourselves imo https://guides.rubyonrails.org/active_job_basics.html#globalid

Reply:
That's ok, but is there any reason for that? Sharing objects with GID seems non-transparent to me, since it just serialize the ID (not the record) in the end. It seems easier to accept id directly to make it clear.

What should we prefer? id or gid? Would be nice to decide this and keep it consistent if possible. :pray:

Reply:
The rails standard way is to pass in the object, and allow the active job serializer / deserializer to do its thing

Reply:
ok, I'll keep it in my mind for future jobs